source/components/Link.tsx

```diff
@@ -0,0 +1,14 @@
+/*
+	This module is used to extend the react-router-dom lib by overriding
+	the Link component to keep the current search parameters when navigating.
+*/
```

Comment:
I'm not sure about this feature, compared to "swallowing" the query param in the state of the app : if I'm not mistaken, the aim is to generalize the use of this component instead of the native Link, hence any search param woul be passed to the next page, right ? Which means that search params relevant to some page (e.g. the "/fin" page) would be passed to pages where they are irrelevant ?

Reply:
Possible solution : define which query params are to stay in the URL permanently, ignore others ?

Reply:
Yes isn't the best way indeed.. I will saved it in the state.

Reply:
After reflexion, I don't really get how to _swallow the query param in the state of the app_. So, I went for the second solution is it ok for you?

Reply:
:100: